File: 88_Merge_Sorted_Array.js
var merge = function (nums1, m, nums2, n) {
  // Step 1: Combine both arrays
  nums1.splice(m, n, ...nums2); // Replace zeros with nums2 elements

  // Step 2: Sort the combined array in increasing order
  nums1.sort((a, b) => a - b);
};

File: 88_Merge_Sorted_Array.py
from typing import List

class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        # Step 1: Combine both arrays
        nums1[m:] = nums2  # Place nums2 elements at the end of nums1
        
        # Step 2: Sort the combined array in increasing order
        nums1.sort()
